tmp.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2018-2020 Streamlit Inc.
```

Comment:
presumably this is a test file that should be deleted before merging?

Reply:
Done